nightingale_manipulation/src/nightingale_manipulation/manipulation_action_client.py

```diff
@@ -70,32 +67,17 @@ def __init__(self):
         )
         self.right_arm.wait_for_server()
 
-        # TODO: replace with service call
-        self.left_arm_joint_names = CFG["left_arm_home"]["names"]
-        self.right_arm_joint_names = CFG["right_arm_home"]["names"]
-        self.left_arm_home_joint_values = CFG["left_arm_home"]["joints"]
-        self.right_arm_home_joint_values = CFG["right_arm_home"]["joints"]
-
-    def cmd_right_arm(self, joint_values, blocking=True):
-        goal = joint_goal(joint_values, self.right_arm_joint_names)
-        self.right_arm.send_goal(goal)
-        if blocking:
-            return self.right_arm.wait_for_result()  # TODO: action_server.get_result()
-        return True
-
-    def cmd_left_arm(self, joint_values, blocking=True):
-        goal = joint_goal(joint_values, self.left_arm_joint_names)
+    def cmd_left_arm(self, goal, blocking=True):
         self.left_arm.send_goal(goal)
         if blocking:
             return self.left_arm.wait_for_result()  # TODO: action_server.get_result()
         return True
 
-    def home(self):
-        self.cmd_left_arm(self.left_arm_home_joint_values, blocking=False)
-        self.left_arm.wait_for_result()
-        self.cmd_right_arm(self.right_arm_home_joint_values, blocking=False)
-        self.right_arm.wait_for_result()
-        return True  # TODO: action_server.get_result()
+    def cmd_right_arm(self, goal, blocking=True):
```

Comment:
im not a fan of this non-blocking functionality. if blocking is set to false, is there any callback or anything that indicates whether or not the maneuver was successful? it seems like if blocking is false we have no indication of what happened.

my advice is just to delete the non-blocking implementation

Reply:
The initial reason for non blocking was so that the left and right arm can move at the same time. I am working on removing this in my PR so just leave it here for now